src/thunderbots/software/util/ros_messages.cpp

```diff
@@ -0,0 +1,17 @@
+#include "util/ros_messages.h"
+
+namespace Util
+{
+    namespace ROSMessages
+    {
+        Ball createBallFromROSMessage(const thunderbots_msgs::Ball& ball_msg)
+        {
+            Point ball_position  = Point(ball_msg.position.x, ball_msg.position.y);
+            Vector ball_velocity = Vector(ball_msg.velocity.x, ball_msg.velocity.y);
+
+            Ball ball = Ball(ball_position, ball_velocity);
+
+            return ball;
+        }
```

Comment:
Yes we could do another layer of namespaces. I think it will be fine for now, as I only expect each message-type to have 2 functions (msg -> class, and class -> msg). Won't change for now but agree it's worth watching.